packages/python/accern_xyme/accern_xyme.py

```diff
@@ -1604,12 +1669,13 @@ def from_node_info(
         return res
 
     def update_info(self, node_info: NodeInfo) -> None:
-        if self._node_id != node_info["id"]:
+        if self.get_id() != node_info["id"]:
             raise ValueError(f"{self._node_id} != {node_info['id']}")
         self._node_name = node_info["name"]
         self._type = node_info["type"]
+        owner = self.as_owner()
```

Comment:
can't assume owner

Reply:
is owner really necessary?
can't we just lazily get the owner when we *actually* need them?